CONTRIBUTING.md

```diff
@@ -28,6 +28,17 @@ cp .env_example .env
 vim .env
 ```
 
+For example if your [api](https://github.com/MysteriumNetwork/api) is listening
```

Comment:
Idea is to `cp .env_example .env` and edit file.
Could You add these recommendations to `.env_example` file